pkg/scheduler/internal/cache/cache.go

```diff
@@ -280,9 +280,11 @@ func (cache *schedulerCache) updateNodeInfoSnapshotList(snapshot *Snapshot, upda
 	if updateAll {
 		// Take a snapshot of the nodes order in the tree
 		snapshot.nodeInfoList = make([]*framework.NodeInfo, 0, cache.nodeTree.numNodes)
-		cache.nodeTree.resetExhausted()
-		for i := 0; i < cache.nodeTree.numNodes; i++ {
-			nodeName := cache.nodeTree.next()
+		nodesList, err := cache.nodeTree.list()
+		if err != nil {
+			klog.Error(err.Error())
```

Comment:
nit: (applies elsewhere)

```suggestion
			klog.Error(err)
```

Reply:
I only found this occurence. which elsewhere are you referring to ? would you mind clarifying ?

Reply:
I noticed the other occurrences in *_test.go.